contracts/VALORTimedMint.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/access/Ownable.sol"; //not 100% necessary I think, just a modifier would suffice as well
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract VALORTimedMint is ERC20, Ownable {
+
+    uint256 public timeUntilNextMint;
+    uint256 public claimPeriod;
+    uint256 private mintCap;
+
+    event Claim(address claimant, uint256 amount);
+    
+    constructor(
+        uint256 _mintCap,
+        uint256 _timeDelay,
+        uint256 mintAmount
+    )   
+        ERC20("VALORIZEDAO", "VALOR") 
```

Comment:
So we want this to be a parameter still because this will be reused as a token template that people call through our interface. I would expect something like:
``` ts
    constructor(
      // other constructor arguments
        string memory name,
        string memory symbol,
    ) ERC20(name, symbol) {
    ```